signaler.py

```diff
@@ -6,34 +6,28 @@
 from time import time, sleep
 
 def invader_detect():
-  """Takes a screen and looks for red clumps"""
-  image = screenshot()
-  red_count = 0
-  pink_count = 0
-  total = 0
+  """Takes a screenshot and looks for red clumps"""
+  image = screenshot(); image.save('temp.png')
+  red_count, pink_count = 0, 0
+  total = 0 # not used
+
   for pixel in image.getdata():
-    total += 1
-    if pixel[0] > 160 and pixel[1] < 55 and pixel[2] < 60:
-      red_count += 1
+    total += 1 # not used
+    if pixel[0] > 100 and pixel[1] < 50:
+      if pixel[2] < 50: red_count += 1
+      elif pixel[2] > 150: pink_count += 1
```

Comment:
The magic numbers here were tuned by hand using trial and error. I'm concerned collapsing them like this will throw off detection (currently rather consistent for reds)

Reply:
I grabbed a number of pictures of red & pink invaders & tested against them when I changed those numbers. My apologies, I should have given more detail on this. I saw no accuracy loss when making this change on the test cases I was using. 

Regardless, please do check if this difference is showing any accuracy loss on your system. I would be curious if so.

Reply:
In that case, I'll try it.

Reply:
@Lord-Vorian 
I ran a better documented test on the new function vs the original. Here is all the data I got from it and how I ran it so you can recreate it if you would like:

The test code has copies of both versions of the function, it only requires pyautogui & time.
[Link to testing code](https://gist.github.com/satetheus/35f4e943f6d31dafb37a915120a880d7)

[Here](https://gist.github.com/satetheus/35f4e943f6d31dafb37a915120a880d7#gistcomment-2921493) is the output I got from the test function. The number of the picture corresponds to the order of these pictures:


![1](https://user-images.githubusercontent.com/37316851/57989934-dd4f7f80-7a5e-11e9-83ac-a2a3e66eca6f.jpg)
![2](https://user-images.githubusercontent.com/37316851/57989935-dd4f7f80-7a5e-11e9-9d22-e60748f0f988.jpg)
![3](https://user-images.githubusercontent.com/37316851/57989936-dd4f7f80-7a5e-11e9-8cdc-6d19a8cf967a.jpg)
![4](https://user-images.githubusercontent.com/37316851/57989937-dd4f7f80-7a5e-11e9-8f0a-0efd882b9578.jpg)
![5](https://user-images.githubusercontent.com/37316851/57989938-dd4f7f80-7a5e-11e9-8437-0ecd4b2c6f59.jpg)
![6](https://user-images.githubusercontent.com/37316851/57989939-dde81600-7a5e-11e9-8b94-d3b5663ad616.jpg)
![7](https://user-images.githubusercontent.com/37316851/57989940-dde81600-7a5e-11e9-96ac-022f3ab544f5.jpg)
![8](https://user-images.githubusercontent.com/37316851/57989941-dde81600-7a5e-11e9-8543-3dea80d32aba.jpg)